documentation/assemblies/assembly-configuration-sizing-guide.adoc

```diff
@@ -5,20 +5,32 @@
 [id='sizing-guide-{context}']
 = {ProductName} configuration sizing guidelines
 
-The sizing guide provides guidelines on how to size {ProductName} installations. More specifically, these guidelines offer specific configuration recommendations for components and plans based on use cases, and the trade-offs involved when adjusting the configuration settings. Sizing {ProductName} involves configuration of:
+The following information provides guidelines on how to size {ProductName} installations. More specifically, these guidelines offer specific configuration recommendations for components and plans based on use cases, and the trade-offs involved when adjusting the configuration settings. Sizing {ProductName} involves configuration of:
 
-* Brokers 
+* Brokers
 * Routers (standard address space only)
 * Operator(s)
 * Plans
 
-Each address space type has some distinct features that should be taken into account when creating the address plans. For more information about address space types and their semantics, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#con-address-space-messaging[address spaces].
+Each address space type has certain distinct features that need to be considered when creating the address plans.
```

Comment:
@jenmalloy Somehow my comment got lost, but this is the sentence I think should be changed. The whole guide is not just about address plans, but also infra configs, address space configs etc. so it read a bit weird to see just address plans specifically here.

Reply:
@lulf ah, ok, understood. How about adding "For example" at the beginning of the sentence; would that help?